CondTools/CTPPS/src/WriteCTPPSPixelAnalysisMask.cc

```diff
@@ -70,10 +67,11 @@ void WriteCTPPSPixelAnalysisMask::analyze(const edm::Event &, edm::EventSetup co
   */
 
   // Write Analysis Mask to sqlite file:
-  const CTPPSPixelAnalysisMask *pCTPPSPixelAnalysisMask = hAnalysisMask.product();  // Analysis Mask
+
+  const auto analysisMask = &es.getData(tokenAnalysisMask_);
   edm::Service<cond::service::PoolDBOutputService> poolDbService;
   if (poolDbService.isAvailable()) {
-    poolDbService->writeOneIOV(*pCTPPSPixelAnalysisMask, analysismaskiov_, /*m_record*/ record_);
+    poolDbService->writeOneIOV(analysisMask, analysismaskiov_, /*m_record*/ record_);
```

Comment:
```suggestion
    poolDbService->writeOneIOV(analysisMask, analysismaskiov_, record_);
```

Reply:
@tvami here I get from @wpcarvalho (who is testing the package) the following comment:   
It is saving a CTPPSPixelDAQMappingconst* object instead of a CTPPSPixelDAQMapping
  Removing "&" in 
     const auto mapping = &es.getData(tokenMapping_);
would fix the issue.
I'm a bit puzzled now...